### ISSUE TITLE
Display default values in dataclass tooltip

### ISSUE THREAD

**Issue by cdce8p** [microsoft/pylance-release#585]
## Environment data

-   Language Server version: 2020.11.0
-   OS and version: macOS
-   Python version: 3.9

## Expected behaviour

As with normal classes, tooltips should display default values.

![Screen Shot 2020-11-08 at 14 01 51](https://user-images.githubusercontent.com/30130371/98465713-0de32600-21cb-11eb-82d6-e7a145aa3e43.png)

## Actual behaviour

![Screen Shot 2020-11-08 at 14 02 10](https://user-images.githubusercontent.com/30130371/98465715-1176ad00-21cb-11eb-8bcd-1a1aa5d1e15d.png)

## Code Snippet / Additional information

```python
from dataclasses import dataclass

class MyClass:
    def __init__(self, var: bool, var_default: bool = False):
        pass

@dataclass
class MyDataClass:
    var: bool
    var_default: bool = False
```

**Comment by erictraut** [comment]
Thanks for the suggestion. This will be addressed in the next release.

**Comment by jakebailey** [comment]
This issue has been fixed in version 2020.11.1, which we've just released. You can find the changelog here: https://github.com/microsoft/pylance-release/blob/master/CHANGELOG.md#2020111-11-november-2020
